apiary.apib

```diff
@@ -577,3 +577,191 @@ API endpoints that Operation Code's Rails backend makes available to its React f
         {
             errors: "Some error message"
         }
+
+## CodeSchool | Collection [/api/v1/code_schools]
```

Comment:
Two things:

1.  Please create a free account at apiary.io, and use their editor to confirm that all of these changes do not create any semantic issues 
2. These endpoints are alphabetized, so these additions will need to go at the top of the file